Ginger/Ginger/AutomatePageLib/NewAutomatePage.xaml.cs

```diff
@@ -643,7 +643,10 @@ void ToggleProcessButtons(bool isEnabled)
 
         void ToggleActivityPageUIButtons(bool IsEnabled)
         {
-            mActivityPage.SetUIElementsBehaverBasedOnRunnerStatus(IsEnabled);
+            if (mActivityPage != null)
```

Comment:
i think we should check Why the page is null. Because in this this function we are enabling the buttons and setting visibility